packages/eslint-plugin-gestalt/src/prefer-heading.test.js

```diff
@@ -0,0 +1,108 @@
+// @flow strict
+import {
+  getPathFormatterByRuleName,
+  getRuleTester,
+  getTestTypePrepender,
+  readTestByPath,
+} from './helpers/testHelpers.js';
+import rule from './prefer-heading.js';
+
+const ruleName = 'prefer-heading';
+const ruleTester = getRuleTester();
+const pathFormatter = getPathFormatterByRuleName(ruleName);
+const validPrepender = getTestTypePrepender('valid');
+const invalidPrependerNoDisallowed = getTestTypePrepender('invalid');
```

Comment:
I'm not sure I understand the "NoDisallowed" suffix here?

Reply:
DONE